lib/foreman_kubevirt/engine.rb

```diff
@@ -4,7 +4,7 @@ class Engine < ::Rails::Engine
     config.autoload_paths += Dir["#{config.root}/app/models/concerns"]
     initializer "foreman_kubevirt.register_plugin", :before => :finisher_hook do |_app|
       Foreman::Plugin.register :foreman_kubevirt do
```

Comment:
Just to be aligned with other plugins, we could probably wrap this in `app.reloader.to_prepare` block.

Reply:
Perhaps that also solves the deprecation warning I still saw. Will try

Reply:
Could you submit that as a change in https://github.com/theforeman/foreman_plugin_template/blob/master/lib/foreman_plugin_template/engine.rb as well?

Reply:
> Perhaps that also solves the deprecation warning I still saw. Will try

That it does.

Reply:
> Could you submit that as a change in https://github.com/theforeman/foreman_plugin_template/blob/master/lib/foreman_plugin_template/engine.rb as well?

https://github.com/theforeman/foreman_plugin_template/pull/67